<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UserTestStatsWindow</class>
 <widget class="QDialog" name="UserTestStatsWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>729</width>
    <height>321</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Просмотр статистики по пройденным тестам - RayTracing</string>
  </property>
  <widget class="QLabel" name="label_user_name">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>141</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt; color:#ffffff;&quot;&gt;Пользователь:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="text_user_name">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>20</y>
     <width>241</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QTableWidget" name="table_stats">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>711</width>
     <height>251</height>
    </rect>
   </property>
   <property name="editTriggers">
    <set>QAbstractItemView::NoEditTriggers</set>
   </property>
   <property name="showGrid">
    <bool>true</bool>
   </property>
   <property name="gridStyle">
    <enum>Qt::SolidLine</enum>
   </property>
   <attribute name="horizontalHeaderDefaultSectionSize">
    <number>167</number>
   </attribute>
   <column>
    <property name="text">
     <string>ID теста</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Тема теста</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Оценка</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Дата прохождения</string>
    </property>
   </column>
  </widget>
  <widget class="QPushButton" name="button_exit">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>10</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Закрыть окно со статистикой по пройденным тестам</string>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    background-color: rgb(255, 255, 255);
    border: 2px solid #094065;
}

QPushButton:pressed {
    background-color:  rgb(118, 118, 118);
}
</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>images/exit-button.png</normaloff>images/exit-button.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>30</width>
     <height>30</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="button_help">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>10</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Получить справку</string>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    background-color: rgb(255, 255, 255);
    border: 2px solid #094065;
}

QPushButton:pressed {
    background-color:  rgb(118, 118, 118);
}
</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>images/help-button.png</normaloff>images/help-button.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>36</width>
     <height>36</height>
    </size>
   </property>
  </widget>
  <widget class="QLabel" name="label_id_user">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>20</y>
     <width>31</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt; color:#ffffff;&quot;&gt;ID:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="text_id_user">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>20</y>
     <width>111</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
